### ISSUE TITLE
Release header permission for javascript interest

### ISSUE THREAD

**Issue by talesam** [joseluisq/static-web-server#115]
I'm using the following configuration in my docker-compose
```
version: "3.9"

services:
  checkuser:
    image: joseluisq/static-web-server:2
    container_name: checkuser
    restart: always
    environment:
#      - SERVER_CORS_ALLOW_ORIGINS=
      - SERVER_CORS_ALLOW_HEADERS="*"
    volumes:
      - /home/ubuntu/Docker/CheckUser/html:/public/checkuser
    ports:
      - 5001:80/tcp
    networks:
      checkuser-net:

networks:
  checkuser-net:
      name: checkuser-net
```

**I would like two things:**
### 1) Make a release similar to this line:
```
erver.headers['Access-Control-Allow-Origin'] = '*'
```

### 2) Display a .json file with no extension.
For example the file calls "**test**" and when I open it in the browser, I click to save, it saves as "**test.json**"

**Comment by joseluisq** [comment]
What do you mean by "release header permission"?

**Comment by joseluisq** [comment]
> ### 1) Make a release similar to this line:
> ```
> erver.headers['Access-Control-Allow-Origin'] = '*'
> ```

If you want to customize headers that the server will respond then have a look at https://sws.joseluisq.net/features/custom-http-headers/

> ### 2) Display a .json file with no extension.
> For example the file calls "**test**" and when I open it in the browser, I click to save, it saves as "**test.json**"

It sounds like you want to customize routes to overwrite requests URI paths. So that feature is not supported but it is on my TODO list.

**Comment by talesam** [comment]
> > ### 1) Make a release similar to this line:
> > ```
> > erver.headers['Access-Control-Allow-Origin'] = '*'
> > ```
> 
> If you want to customize headers that the server will respond then have a look at https://sws.joseluisq.net/features/custom-http-headers/
> 
> > ### 2) Display a .json file with no extension.
> > For example the file calls "**test**" and when I open it in the browser, I click to save, it saves as "**test.json**"
> 
> It sounds like you want to customize routes to overwrite requests URI paths. So that feature is not supported but it is on my TODO list.

This is exactly what I needed, so it will be implemented yet. Thanks!

**Comment by joseluisq** [comment]
Relates to #116

**Comment by joseluisq** [comment]
Custom URL rewrites released on [v2.10.0](https://github.com/joseluisq/static-web-server/releases/tag/v2.10.0)
